src/scripts/get_era5_data.py

```diff
@@ -0,0 +1,164 @@
+    print('python get_erainterim_data.py sfc 2015-09-29 00/06/12/18')
+    exit(0)
+
+
+if __name__ == "__main__":
+
+    # data_basedir = "/misc/pearl/extdata/ERA-Interim/"
+    data_basedir = "/ecmwf-data/"
+
+    if len(argv) < 4:
+        usage()
+
+    levtype = argv[1]
+    if levtype == 'pl':
+        cmd = cmd_pl
+    elif levtype == 'sfc':
+        cmd = cmd_sfc
+    else:
+        usage()
+
+    hours = argv[3].split('/')
+    for hour in hours:
+        if hour not in ['00', '06', '12', '18']:
+            usage()
+
+    date_start = datetime(2001, 8, 1).date()
+    # data time delay is three months
+    date_end = datetime.now().date() - relativedelta(months=2)
+    date_end = (
+        datetime(date_end.year, date_end.month, 1) - timedelta(days=1)
+    ).date()
+    n = 1000
+
+    dates = []
+    if argv[2] == '?':
+        # retrieve data from n days
+        ni = 0
+        i = 0
+        while ni < n:
+            candidate_date = date_end - timedelta(days=i)
+            # check if file already exists
+            year = "{}".format(candidate_date.year)
+            month = "{0:02}".format(candidate_date.month)
+            target_dir = os.path.join(data_basedir, year, month)
+            date_string = str(candidate_date)[0: 10]
+            file_already_exists = 0
+            for hour in hours:
+                target_file = "{}_{}_{}-{}.nc".format(
+                    cmd['class'], cmd['levtype'], date_string, hour,
+                )
+                fullfile = os.path.join(target_dir, target_file)
+                if os.path.exists(fullfile):
+                    file_already_exists = 1
+            i = i + 1
+            if candidate_date >= date_start and not file_already_exists:
+                ni = ni + 1
+                dates.append(candidate_date)
+            if candidate_date < date_start:
+                break
+    else:
+        try:
+            dates.append(datetime.strptime(argv[2], '%Y-%m-%d'))
+        except Exception:
+            usage()
+
+    # loop over dates
+    for date in dates:
+        year = "{0}".format(date.year)
+        month = "{0:02}".format(date.month)
+        date_string = str(date)[0: 10]
+        cmd['date'] = date_string
+
+        # create target directory if not exists
+        target_dir = os.path.join(data_basedir, year, month)
+        if not os.path.exists(target_dir):
+            os.makedirs(target_dir)
+        # loop ove hours
+        for hour in hours:
+
+            cmd['time'] = hour
+            target_file = "{}_{}_{}-{}.nc".format(
+                cmd['class'], cmd['levtype'], cmd['date'], cmd['time'],
+            )
+            target = os.path.join(target_dir, target_file)
+            print(target)
+
+            # check if desired file already exists
+            # retrieve data if file not exists
+            if not os.path.exists(target):
+                pass
+                server = cdsapi.Client()
+                server.retrieve(service,cmd,target)
```

Comment:
should be something like:
dataset = "reanalysis-era5-pressure-levels"
server.retrieve(dataset, cmd, target) 
according to https://cds.climate.copernicus.eu/api-how-to

have verified that files are retrieved with these changes